public/app/features/browse-dashboards/components/RecentlyViewedDashboards.tsx

```diff
@@ -31,11 +46,19 @@ export function RecentlyViewedDashboards() {
     <CollapsableSection
       headerDataTestId="browseDashboardsRecentlyViewedTitle"
       label={
-        <Text variant="h5" element="h3">
-          <Trans i18nKey="browse-dashboards.recently-viewed.title">Recently viewed</Trans>
-        </Text>
+        <Stack direction="row" justifyContent="space-between" alignItems="baseline" width="100%">
+          <Text variant="h5" element="h3" onClick={() => setIsOpen(!isOpen)}>
+            <Trans i18nKey="browse-dashboards.recently-viewed.title">Recently viewed</Trans>
+          </Text>
+          <Button icon="times" size="xs" variant="secondary" fill="text" onClick={handleClearHistory}>
+            {t('browse-dashboards.recently-viewed.clear', 'Clear history')}
+          </Button>
+        </Stack>
       }
-      isOpen={true}
+      isOpen={isOpen}
```

Comment:
We also have ControlledCollapsible section, which handles the open state internally, maybe it'd work here a well?